charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -189,6 +189,8 @@ spec:
           {{- if .Values.clusterAgent.admissionController.enabled }}
           - name: DD_ADMISSION_CONTROLLER_ENABLED
             value: {{ .Values.clusterAgent.admissionController.enabled | quote }}
+          - name: DD_ADMISSION_CONTROLLER_WEBHOOK_NAME
+            value: {{ .Values.clusterAgent.admissionController.webhookName | quote }}
```

Comment:
This parameter doesn’t seem defined in the `values.yaml` file: https://github.com/DataDog/helm-charts/blob/bb9de14bf5bf070f55cec896cd22cfd1e1fac83e/charts/datadog/values.yaml#L951-L978

Reply:
I forgot to save the file. It should be fixed now

Reply:
do we know it the `cluster-agent` is able to unregister previous webhook if it doesn't use it anymore? (exemple if the webhook is renamed due to this new setting)